dialdbg/src/main.rs

```diff
@@ -121,6 +122,48 @@ async fn dial_webrtc(
     }
 }
 
+const MDNS_SERVICE_NAME: &'static str = "_rpc._tcp.local";
```

Comment:
since we're already using values from `viam::rpc::dial`, can we just use the `SERVICE_NAME` const from there instead of recreating it here?